movement/movement_custom_signal_tunes.h

```diff
@@ -84,4 +84,21 @@ int8_t signal_tune[] = {
 };
 #endif // SIGNAL_TUNE_KIM_POSSIBLE
 
+#ifdef SIGNAL_TUNE_LAYLA
+int8_t signal_tune[] = {
+    BUZZER_NOTE_A6, 4,
+    BUZZER_NOTE_REST, 1,
+    BUZZER_NOTE_C7, 5,  
+    BUZZER_NOTE_REST, 2,
+    BUZZER_NOTE_D7, 5,
+    BUZZER_NOTE_REST, 1,
+    BUZZER_NOTE_F7, 5,
+    BUZZER_NOTE_REST, 1,
+    BUZZER_NOTE_D7, 5,
+    BUZZER_NOTE_REST, 2,
```

Comment:
Another C7 is missing between two D7s.

Reply:
thanks for pointing this out! I was wondering why last notes part sounded off